doc/api/sqlite.md

```diff
@@ -77,20 +77,24 @@ console.log(query.all());
 
 <!-- YAML
 added: v22.5.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/56991
+    description: The `path` argument now supports Buffer and URL objects.
 -->
 
 This class represents a single [connection][] to a SQLite database. All APIs
 exposed by this class execute synchronously.
 
-### `new DatabaseSync(location[, options])`
+### `new DatabaseSync(path[, options])`
 
 <!-- YAML
 added: v22.5.0
 -->
 
-* `location` {string} The location of the database. A SQLite database can be
+* `path` {string | Buffer | URL} The path of the database. A SQLite database can be
```

Comment:
I think you should mention that an URI filename also works and link to https://www.sqlite.org/uri.html.

Reply:
It has landed already. Do you want to add it (in a new PR?)

Reply:
I still have another SQLite PR to get to 🙈 

Maybe after that one if you didn't beat me to it!

Reply:
Good!

Thanks for all your comments.

Reply:
Thanks for addressing them